src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -51,7 +51,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
 
         return new Configuration(
             $container->getParameter('kernel.debug'),
-            $container->getParameter('kernel.project_dir')
+            $container->getParameter('kernel.project_dir'),
```

Comment:
This should be `kernel.root_dir`

Reply:
Changed in be52c2c8b0b316fc47ee956df5fa1013f26a1b7b.